website/server/controllers/api-v3/groups.js

```diff
@@ -366,7 +366,7 @@ api.rejectGroupInvite = {
  * @apiGroup Group
  *
  * @apiParam {String} groupId The group _id ('party' for the user party and 'habitrpg' for tavern are accepted)
- * @apiParam {String="remove-all","keep-all"} keep Query parameter - Whether to keep or not challenges' tasks. Defaults to keep-all
+ * @apiParam {String="remove-all","keep-all"} keep Query parameter - Whether or not to keep challenges belonging to the group being left. Defaults to keep-all
```

Comment:
This should follow the APIDoc way of designating a default param: http://apidocjs.com/#param-api-param and use the Query param group (newly created)

```js
* @apiParam (Query) {String="remove-all","keep-all"} keep=keep-all  Whether or not to keep challenges belonging to the group being left.
```